### ISSUE TITLE
0.1.5 AllTheOres Recipes Missing 

### ISSUE THREAD

**Issue by MalokXV** [AllTheMods/ATM-9#421]
Noticed this due to compacting drawers, recipes for Copper and Zinc nuggets from AllTheOres are missing. Copper and Zinc are now crafted into nuggets from Create, JEI states that the recipe is from AllTheOres. As I was writing this realized that the zinc ingots/blocks are also not working correctly? as smelting Raw Zinc from AllTheOres turns it into the ingots from Create as well. Copper ingots seem to be ok though.

TLDR: Zinc from ATO is messed up, and Copper nuggets from ATO are messed up.

**Comment by TheBedrockMaster** [comment]
Is this still like this?
